packages/headless/src/controllers/commerce/core/facets/date/headless-commerce-date-facet.ts

```diff
@@ -15,6 +15,8 @@ import {
   buildCoreCommerceFacet,
 } from '../headless-core-commerce-facet.js';
 
+export type {DateFacetValue};
+
```

Comment:
These facet Value interface exports are here because not all exports of Facet Value were created equal. In the sample, I was importing some from @coveo/headless/commerce and some from @coveo/headless/ssr-commerce. Now they all are imported from @covoe/headless/ssr-commerce.